integ-test/src/test/resources/correctness/queries/filter.txt

```diff
@@ -0,0 +1,5 @@
+SELECT AVG(AvgTicketPrice) FILTER(WHERE Carrier = 'Kibana Airlines') AS filtered FROM kibana_sample_data_flights
```

Comment:
Could you add test case to check if we can support function call in `WHERE`? Just want to check if pushdown works in this case.